packages/get-starknet/webpack.config.dev.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
```

Comment:
add // eslint-disable-next-line @typescript-eslint/no-var-requires for each require